tempo.yaml

```diff
@@ -0,0 +1,35 @@
+server:
+  http_listen_port: 3200
+
+distributor:
+  receivers:
+    otlp:
+      protocols:
+        grpc:
+        http:
+
+ingester:
+  trace_idle_period: 10s
+  max_block_duration: 5m
+
+compactor:
+  compaction:
+    block_retention: 1h
+
+metrics_generator:
+  registry:
+    external_labels:
+      source: tempo
+
+storage:
+  trace:
+    backend: local
+    wal:
+      path: /tmp/tempo/wal
+    local:
+      path: /tmp/tempo/traces
+
+overrides:
+  max_bytes_per_trace: 1000000000  # ~100 MB per trace limit
+  ingestion_rate_limit_bytes: 60000000    # 60MB/sec
+  ingestion_burst_size_bytes: 120000000   # 120MB burst
```

Comment:
Oh i should revisit these overrides, this shouldn't be needed now we've removed all the old spans.